sdk/communication/Azure.Communication.CallingServer/src/Models/CallMediaRecognizeOptions.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+
+namespace Azure.Communication.CallingServer
+{
+    /// <summary>
+    /// The Call Media Recognize Options.
+    /// </summary>
+    public abstract class CallMediaRecognizeOptions
+    {
+        /// <summary>
+        /// Creates a new instance of the CallMediaRecognizeOptions.
+        /// </summary>
+        /// <param name="inputType"></param>
+        /// <param name="targetParticipant"></param>
+        protected CallMediaRecognizeOptions(RecognizeInputType inputType, CommunicationIdentifier targetParticipant)
+        {
+            InputType = inputType;
+            TargetParticipant = targetParticipant;
+        }
+
+        /// <summary>
+        /// Recognize Input Type.
+        /// </summary>
+        public RecognizeInputType InputType { get; }
+
+        /// <summary>
+        /// Should stop current Operations?.
```

Comment:
I think we should update this to a statement instead of question, and change this to "Interrupt current call media operation". ?